src/components/NonProject/NonProject.js

```diff
@@ -1,17 +1,52 @@
-import React from 'react'
-import './NonProject.css'
+import React from "react"
+import ReactDOM from "react-dom"
 
-function Joke(props) {
-	console.log(props)
-	return (
-	    <div className="joke">
-		        <h3 style={{display: props.question ? "block" : "none"}}>
-		         	Question: {props.question} 
-		        </h3>
-		        <h3 style={{color: !props.question && "blue"}}> Answer: {props.punchline} </h3>
-		    	<hr/>            
-        </div>
-	)
+// #1
```

Comment:
Please avoid comments. Try to write self-explainable and well readable code so in 98% of cases you do not need comments.

Reply:
I know. It was in the lesson code, so i left it. Sorry

Reply:
Fixed.